schemas/applicants/applicant_builder.yaml

```diff
@@ -0,0 +1,3 @@
+allOf:
```

Comment:
In the existing php lib we called it `ApplicantRequest` instead of `Builder`. https://github.com/onfido/api-php-client/blob/master/lib/Model/ApplicantRequest.php

Reply:
I dont have a strong opinion about it, but I remember seeing the `Request` suffix in other client libs in the past